src/SFA.DAS.Commitments.Api.Types/Commitment/CommitmentView.cs

```diff
@@ -20,9 +20,7 @@ public CommitmentView()
         public string LegalEntityName { get; set; }
         public long? ProviderId { get; set; }
         public string ProviderName { get; set; }
-        public long? TransferSenderId { get; set; }
-        public string TransferSenderName { get; set; }
-        public TransferApprovalStatus TransferApprovalStatus { get; set; }
+        public TransferSenderInfo TransferSenderInfo { get; set; }
```

Comment:
Suffixing "Info" is superfluous. We don't have ApprenticeshipInfo, or CommitmentInfo!

Reply:
Done for Id and Name